meta-facebook/yv35-rf/src/platform/plat_class.h

```diff
@@ -0,0 +1,25 @@
+#ifndef PLAT_CLASS_H
+#define PLAT_CLASS_H
+
+#include "hal_gpio.h"
+#include "plat_gpio.h"
+
+enum BOARD_ID {
+	RAINBOW_FALLS = 0x0A,
+	WAIMANO_FALLS = 0x0C,
+	VERNAL_FALLS = 0x0E,
```

Comment:
Vernal Falls is a Yv3 expansion board, isn't it?
I think Yv3.5 E.1S is going to be `KAHUNA_FALLS`

Reply:
@GoldenBug  Hi Peter, this is real VERNAL_FALLS with Aspeed BIC (ODMB is working on this?) 
We haven't added the board id for KAHUNA_FALLS.